### PR TITLE
Moved download to server side.

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

* Moved download to server side.

## TODO

- [x] Tests
- [ ] Implement a better dropdown menu. (Current one is based on style guide.)

## Issues addressed

* Client-side solution for downloading files was not reliable.

### CODE REVIEW

File: kolibri/content/views.py [hunk 1]
```diff
@@ -58,3 +58,32 @@ def get(self, request, zipped_filename, embedded_filepath):
...
+        # generate a file response
+        response = FileResponse(open(path, 'rb'))
+
+        # set the content-type by guessing from the filename
+        response['Content-Type'] = mimetypes.guess_type(filename)[0]
+
+        # set the content-disposition as attachment to force download
+        response['Content-Disposition'] = 'attachment;'
+
+        # set the content-length tht the file size
```
Comment (jamalex) [inline kolibri/content/views.py]: tht -> to?

File: kolibri/core/assets/src/vue/content-renderer/download-button.vue [hunk 2]
```diff
@@ -1,93 +1,75 @@
 <template>
 
-  <icon-button @click="downloadContent" :text="downloadMediaText">
-    <svg src="download.svg"></svg>
-  </icon-button>
+  <div class="dropdown">
+    <button>Download Content &#9660</button>
```
Comment (jamalex) [inline kolibri/core/assets/src/vue/content-renderer/download-button.vue]: String should be in `$trs`, and then referenced here, so it will get translated.

Comment (christianmemije) [reply kolibri/core/assets/src/vue/content-renderer/download-button.vue]: Fixed.

**Comment by jamalex** [comment]
Looks like there are still some failing tests. Much strict!

**Comment by christianmemije** [comment]
Importing le-utils from pypi should fix this. BRB.

**Comment by christianmemije** [comment]
@jamalex MERGE!!!

**Pull request merged**